CHANGELOG.md

```diff
@@ -1,15 +1,31 @@
-# 1.0.x (unreleased)
+# 1.0.0-RC1
 
 ## New Features
 
 - Added GET subject versions to the cached schema registry client.
+- References librdkafka.redist [1.0.0-RC8](https://github.com/edenhill/librdkafka/releases/tag/v1.0.0-RC8)
+  - supports apline linux out-of-the-box.
```

Comment:
typo, should be appalachian linux

Reply:
ahh, good point. have you also reported this here?: https://alpinelinux.org/